server.js

```diff
@@ -17,6 +25,31 @@ app.get('/', function (req, res) {
     res.send('Welcome');
   });
 
+
+
+// connect to the MongoDB with mongoose
+require('./config/database');
+require("./config/passport");
+
+//passport middleware
+  app.use(
+    session({
+      secret: "FitnessRocks!",
+      resave: false,
+      saveUninitialized: true,
+    })
+  );
+
+  app.use(passport.initialize());
+  app.use(passport.session());
+
+  
+  app.use('/', indexRoutes);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7812747)
```suggestion
  app.use("/", indexRoutes);
```